File: src/DA_LRV_2023/app/Models/group_user.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Kyslik\ColumnSortable\Sortable;

class group_user extends Model
{
    use HasFactory, Sortable;
    protected $table = 'group_user';
    protected $fillable = [
        'id_group',
        'id_user',
        'vai_tro'
        //biet danh
    ];
}

File: src/da_tn_2023_appthongbao/lib/model/group_user.dart
class group_user {
  int? id;
  int? idGroup;
  int? idUser;
  String? vaiTro;
  int? trangThai;

  group_user({this.id, this.idGroup, this.idUser, this.vaiTro, this.trangThai});

  group_user.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    idGroup = json['id_group'];
    idUser = json['id_user'];
    vaiTro = json['vai_tro'];
    trangThai = json['trang_thai'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['id_group'] = this.idGroup;
    data['id_user'] = this.idUser;
    data['vai_tro'] = this.vaiTro;
    data['trang_thai'] = this.trangThai;
    return data;
  }
}